src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/evidenceshare/IssueDirectionFunctionalTest.java

```diff
@@ -33,4 +58,22 @@ public void processAnIssueDirectionEvent_shouldUpdateInterlocReviewState() throw
 
         assertNull(caseData.getDirectionTypeDl());
     }
+
+    @Test
+    public void processAnIssueDirectionEvent_ifPastHearingExcludedDatesAreOnCaseDetails() throws IOException {
```

Comment:
is this really a scenario that needs a functional test, is it not covered in unit/integration tests?